Python_Module/MangDang/LCD/ST7789.py

```diff
@@ -1,180 +1,161 @@
-ST7789_INVON       = 0x21
-ST7789_GAMSET      = 0x26
-ST7789_DISPOFF     = 0x28
-ST7789_DISPON      = 0x29
-ST7789_CASET       = 0x2A
-ST7789_RASET       = 0x2B
-ST7789_RAMWR       = 0x2C
-ST7789_RAMRD       = 0x2E
-
-ST7789_PTLAR       = 0x30
-ST7789_VSCRDEF     = 0x33
-ST7789_TEOFF       = 0x34
-ST7789_TEON        = 0x35
-ST7789_MADCTL      = 0x36
-ST7789_VSCRSADD    = 0x37
-ST7789_IDMOFF      = 0x38
-ST7789_IDMON       = 0x39
-ST7789_COLMOD      = 0x3A
-ST7789_RAMWRC      = 0x3C
-ST7789_RAMRDC      = 0x3E
-
-ST7789_TESCAN      = 0x44
-ST7789_RDTESCAN    = 0x45
-
-ST7789_WRDISBV     = 0x51
-ST7789_RDDISBV     = 0x52
-ST7789_WRCTRLD     = 0x53
-ST7789_RDCTRLD     = 0x54
-ST7789_WRCACE      = 0x55
-ST7789_RDCABC      = 0x56
-ST7789_WRCABCMB    = 0x5E
-ST7789_RDCABCMB    = 0x5F
-
-ST7789_RDABCSDR    = 0x68
-
-ST7789_RDID1       = 0xDA
-ST7789_RDID2       = 0xDB
-ST7789_RDID3       = 0xDC
-
-ST7789_RAMCTRL     = 0xB0
-ST7789_RGBCTRL     = 0xB1
-ST7789_PORCTRL     = 0xB2
-ST7789_FRCTRL1     = 0xB3
-
-ST7789_GCTRL       = 0xB7
-ST7789_DGMEN       = 0xBA
-ST7789_VCOMS       = 0xBB
-
-ST7789_LCMCTRL     = 0xC0
-ST7789_IDSET       = 0xC1
-ST7789_VDVVRHEN    = 0xC2
-
-ST7789_VRHS        = 0xC3
-ST7789_VDVSET      = 0xC4
-ST7789_VCMOFSET    = 0xC5
-ST7789_FRCTR2      = 0xC6
-ST7789_CABCCTRL    = 0xC7
-ST7789_REGSEL1     = 0xC8
-ST7789_REGSEL2     = 0xCA
-ST7789_PWMFRSEL    = 0xCC
-
-ST7789_PWCTRL1     = 0xD0
-ST7789_VAPVANEN    = 0xD2
-ST7789_CMD2EN      = 0xDF5A6902
-ST7789_PVGAMCTRL   = 0xE0
-ST7789_NVGAMCTRL   = 0xE1
-ST7789_DGMLUTR     = 0xE2
-ST7789_DGMLUTB     = 0xE3
-ST7789_GATECTRL    = 0xE4
-ST7789_PWCTRL2     = 0xE8
-ST7789_EQCTRL      = 0xE9
-ST7789_PROMCTRL    = 0xEC
-ST7789_PROMEN      = 0xFA
-ST7789_NVMSET      = 0xFC
-ST7789_PROMACT     = 0xFE
-
-# Colours for convenience
-ST7789_BLACK       = 0x0000 # 0b 00000 000000 00000
-ST7789_BLUE        = 0x001F # 0b 00000 000000 11111
-ST7789_GREEN       = 0x07E0 # 0b 00000 111111 00000
-ST7789_RED         = 0xF800 # 0b 11111 000000 00000
-ST7789_CYAN        = 0x07FF # 0b 00000 111111 11111
-ST7789_MAGENTA     = 0xF81F # 0b 11111 000000 11111
-ST7789_YELLOW      = 0xFFE0 # 0b 11111 111111 00000
-ST7789_WHITE       = 0xFFFF # 0b 11111 111111 11111
-
-
-def color565(r, g, b):
-    """Convert red, green, blue components to a 16-bit 565 RGB value. Components
-    should be values 0 to 255.
-    """
-    return ((r & 0xF8) << 8) | ((g & 0xFC) << 3) | (b >> 3)
-
-def image_to_data(image):
-    """Generator function to convert a PIL image to 16-bit 565 RGB bytes."""
-    # NumPy is much faster at doing this. NumPy code provided by:
-    # Keith (https://www.blogger.com/profile/02555547344016007163)
-    pb = np.array(image.convert('RGB')).astype('uint16')
-    color = ((pb[:,:,0] & 0xF8) << 8) | ((pb[:,:,1] & 0xFC) << 3) | (pb[:,:,2] >> 3)
-    return np.dstack(((color >> 8) & 0xFF, color & 0xFF)).flatten().tolist()
+
+import spidev
+import RPi.GPIO as GPIO
+
+
+__version__ = '0.0.4'
```

Comment:
```suggestion
```

Is this version definition requried?

Reply:
no,I didn't find it is used, so delete it, thanks